frontend/src/components/teamsAndOrgs/messages.js

```diff
@@ -237,6 +237,10 @@ export default defineMessages({
     id: 'management.organisations.stats.cost.estimation',
     defaultMessage: 'Estimated cost by the end of {year}',
   },
+  discountedLabel: {
+    id: 'management.organisations.stats.cost.estimation.discounted',
+    defaultMessage: ' (discounted)',
```

Comment:
I don't like to include trailing spaces in the strings, because if the translator doesn't include it, we will have problems, maybe we should remove the parenthesis too.
On the component you can use `{' '}` or a padding with CSS.